saiph/reduction/famd.py

```diff
@@ -325,6 +325,7 @@ def get_variable_contributions(
     Returns:
         tuple of contributions and cos2.
     """
+    model.row_weights = get_uniform_row_weights(len(df))
```

Comment:
If we want to compute the contributions of a df based on a model computed with another df, we need to adjust row.weights with this new df 
(row_weights are always 1/len(df) for each row)